src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table').children;
+
+for (const rows of table) {
+  const row = rows.querySelectorAll('tr');
```

Comment:
you shouldn't do it on each iteration. it's enough to find all `tr` once not using a loop.